core/os/os_linux.odin

```diff
@@ -894,7 +895,10 @@ get_env :: proc(key: string, allocator := context.allocator) -> (value: string)
 set_env :: proc(key, value: string) -> Errno {
 	runtime.DEFAULT_TEMP_ALLOCATOR_TEMP_GUARD()
 	s := strings.concatenate({key, "=", value, "\x00"}, context.temp_allocator)
-	res := _unix_putenv(strings.unsafe_string_to_cstring(s))
+  key_cstring := strings.unsafe_string_to_cstring(strings.concatenate({key, "\x00"}, context.temp_allocator))
```

Comment:
`odin check -vet` says `s` is unused. Also, please use tabs to indent the code. Your additions are not aligned properly.

Reply:
Fixed